applications/samples/001_validation/source.cpp

```diff
@@ -17,6 +17,8 @@
 
 void callback(llri::message_severity severity, llri::message_source source, const char* message, void* userData)
 {
+    (void)userData;
+    
```

Comment:
I would recommend using [[maybe_unused]] instead, using (void) is technically compiler specific, eventhough supported by most compilers